sdk/src/azure/core/az_mqtt5_rpc_server_hfsm.c

```diff
@@ -244,6 +244,8 @@ AZ_INLINE az_result _handle_request(az_mqtt5_rpc_server* this_policy, az_mqtt5_r
             .request_data = data->payload,
             .request_topic = data->topic,
             .content_type = content_type_str,
+            .command_name = req_event_data->command_name,
```

Comment:
Does this need to be copied/will there be any issues with the span getting deallocated?

Reply:
These data are only valid in the scope of the callback. The target of the callback must copy the information if they need it. After cb and _handle_request return this data is disregarded.